zookeeper-jute/src/main/java/org/apache/jute/BinaryInputArchive.java

```diff
@@ -85,6 +85,14 @@ public String readString(String tag) throws IOException {
     }
     
     static public final int maxBuffer = Integer.getInteger("jute.maxbuffer", 0xfffff);
+    static public int readExtraSize = Integer.getInteger("zookeeper.jute.maxbuffer.extrasize",
```

Comment:
please make it final

Reply:
variable readExtraSize is reinitialized at line number 93.  So it think it cannot be marked final.